adb_graphics/figures/maps.py

```diff
@@ -372,6 +378,34 @@ def _draw_contours(self, ax, not_labeled):
                     print(f'Cannot add contour labels to map for {self.field.short_name} \
                             {self.field.level}')
 
+    def _draw_scatter(self, ax):
+
+        ''' Plot dots at locations on the map that meet a threshold. '''
+
+        field = self.field
+        levels = self.field.clevs
+        alpha_val = 0.75 # set opacity
+        vals = self.field.values()
+        colors = ['white', 'lightskyblue', 'darkblue', 'green', 'darkorange', \
+                  'indianred', 'firebrick']
+
+        ci = copy.copy(vals)
+        ci = np.full_like(ci, colors[0], dtype='object')
+        ci = np.where((vals > levels[0]) & (vals <= levels[1]), colors[1], ci)
+        ci = np.where((vals > levels[1]) & (vals <= levels[2]), colors[2], ci)
+        ci = np.where((vals > levels[2]) & (vals <= levels[3]), colors[3], ci)
+        ci = np.where((vals > levels[3]) & (vals <= levels[4]), colors[4], ci)
+        ci = np.where((vals > levels[4]) & (vals <= levels[5]), colors[5], ci)
```

Comment:
A fun python syntax trick:

```
levels[3] < vals <= levels[4]
```

A loop might also be a good thing here.
```
ci = np.full_like(ci, colors[0], dtype='object')
for i, color in enumerate(colors):
    ci = np.where(levels[i] < vals < levels[i+1], color[i+1], ci)
```

Reply:
The loop is a good idea, and I've tried the loop you suggested.  It worked up until the last iteration, and then crashes because of referencing levels[i+1] which is outside of the levels array.  I had to change it to account for this.  I changed the loop to use levels instead of colors, and added an exception for the last iteration.  The enumerate part caused a problem in pylint, since I never used the "level" variable in "for level in enumerate(levels)".  I changed it to just use the length of the levels array.  The "fun python trick" didn't work, as it complained that I needed to use vals.any or vals.all instead of just vals, and the results weren't correct in either case.